dist/rpm/os-autoinst.spec

```diff
@@ -62,12 +62,23 @@ Source0:        %{name}-%{version}.tar.xz
 %else
 %define yamllint_requires %{nil}
 %endif
+%if 0%{?suse_version} >= 1550
+%bcond_without black
+%else
+%bcond_with black
+%endif
+%if %{with black}
+# The following line is generated from dependencies.yaml
+%define python_style_requires python3-black
+%else
+%define python_style_requires %{nil}
+%endif
 # The following line is generated from dependencies.yaml
 %define test_base_requires %main_requires cpio perl(Benchmark) perl(Devel::Cover) perl(FindBin) perl(Pod::Coverage) perl(Test::Fatal) perl(Test::Mock::Time) perl(Test::MockModule) perl(Test::MockObject) perl(Test::Mojo) perl(Test::Most) perl(Test::Output) perl(Test::Pod) perl(Test::Strict) perl(Test::Warnings) >= 0.029 procps python3-setuptools qemu qemu-tools qemu-x86
 # The following line is generated from dependencies.yaml
 %define test_version_only_requires perl(Mojo::IOLoop::ReadWriteProcess) >= 0.28
 # The following line is generated from dependencies.yaml
-%define test_requires %build_requires %spellcheck_requires %test_base_requires %yamllint_requires perl(Inline::Python) perl(YAML::PP)
+%define test_requires %build_requires %python_style_requires %spellcheck_requires %test_base_requires %yamllint_requires perl(Inline::Python) perl(YAML::PP)
```

Comment:
I don't think we need to run style checks during the package build. Better add this only to `devel_requires` like it is done for `perl(Perl::Tidy)`.

Reply:
tru, thanks